src/DIRAC/Resources/Computing/ARCComputingElement.py

```diff
@@ -189,7 +189,7 @@ def _writeXRSL(self, executableFile, inputs=None, outputs=None, executables=None
         :param str/list outputs: path of the outputs that we want to get at the end of the execution
         :param str/list executables: path to inputs that should have execution mode on the remote worker node
         """
-        diracStamp = makeGuid()[:8]
+        diracStamp = makeGuid()[:32]
```

Comment:
Shouldn't one strip the hyphens to really get the 32 characters?

Reply:
TBH this could just use a different (simpler) implementation, e.g.

```python
import uuid
diracStamp = uuid.uuid4().hex
```

or 
```
''.join(random.choices(string.ascii_uppercase + string.ascii_lowercase + string.digits, k=32))
```

I can't say which one would be "best", but clearly here we don't necessarily need a GUID.